packages/views/search.py

```diff
@@ -14,6 +15,8 @@
 
 
 class PackageSearchForm(forms.Form):
+    limit = forms.CharField(required=False)
+    page = forms.CharField(required=False)
```

Comment:
Would be nice for these to be proper `IntegerField`s, but they cannot have a default value, so even if you mark them as not required we'll get an exception raised when we try to use the value.

Additionally since we're on an old version we can't specify a default value for `CharField`, but once archweb is finally upgraded to newest Django we could change that to have the defaults specified here.

Reply:
Too bad, but hopefully we will update to a more modern django this year.